src/AppBundle/Controller/LocaleController.php

```diff
@@ -10,14 +10,12 @@
  * file that was distributed with this source code.
  */
 
-namespace AppBundle\Controller\Language;
+namespace AppBundle\Controller;
 
-use AppBundle\Controller\BaseController;
 use AppBundle\Model\User\DTO\LocaleSwitcherDTO;
 use FOS\RestBundle\Controller\Annotations as Rest;
 use Nelmio\ApiDocBundle\Annotation\ApiDoc;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
-use Symfony\Component\HttpFoundation\Cookie;
 use Symfony\Component\HttpFoundation\Response;
```

Comment:
the response use statement is not needed anymore, the return typehint must be removed, too.